java-streams-2/src/main/java/com/baeldung/breakforeach/CustomForEach.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.breakforeach;
+
+import java.util.Spliterator;
+import java.util.function.BiConsumer;
+import java.util.stream.Stream;
+
+public class CustomForEach {
+
+    static class Breaker {
+        private boolean shouldBreak = false;
+
+        void breakForEach() {
```

Comment:
Let's rename this `break`

Reply:
It's not possible. `break` is a keyword. Do you have another suggestion?